jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -426,12 +471,12 @@ public boolean sweep()
     {
         pool.values().stream().filter(entry -> entry.getPooled() instanceof Sweeper.Sweepable).forEach(entry ->
```

Comment:
The `filter()` lambda is wrong, it's `entry.getPooled.connection` that may implement `Sweepable`.
Perhaps it's better to map to `connection` first, then filter, then `forEach()`

Reply:
Good catch, let me do that.